pbf/models.py

```diff
@@ -313,18 +324,11 @@ def rot_converted_this_turn(self):
     def __str__(self):
         return str(self.game.id) + ' - ' + str(self.spirit.name)
 
+    def has_spirit_specific_resource(self):
+        return spirit_to_spirit_specific_resource_name_map.__contains__(self.full_name())
```

Comment:
Why doesn't `self.full_name() in spirit_to_spirit_specific_resource_name_map` work here, that required the use of `__contains__`?

Reply:
Yep, `in` worked!